src/mono/mono/mini/interp/mintops.def

```diff
@@ -705,6 +705,7 @@ OPDEF(MINT_MONO_NEWOBJ, "mono_newobj", 2, Pop0, Push1, MintOpClassToken)
 OPDEF(MINT_MONO_RETOBJ, "mono_retobj", 1, Pop1, Push0, MintOpNoArgs)
 OPDEF(MINT_MONO_ATOMIC_STORE_I4, "mono_atomic.store.i4", 1, Pop2, Push0, MintOpNoArgs)
 OPDEF(MINT_MONO_MEMORY_BARRIER, "mono_memory_barrier", 1, Pop0, Push0, MintOpNoArgs)
+OPDEF(MINT_MONO_EXCHANGE_I8, "mono_interlocked_exchange_long", 1, Pop2, Push1, MintOpNoArgs)
```

Comment:
nit: `"mono_interlocked.exchange.i8"`  (really the only part of this bikeshedding that I feel is somewhat important is to use "i8" instead of "long")

Reply:
Fixed